sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -599,6 +599,70 @@ def get_test_params(self):
         return [params1]
 
 
+class IntervalWidth(_BaseProbaForecastingErrorMetric):
+    """Interval width for interval predictions, sometimes also known as calibration.
```

Comment:
Do you mean calibration or sharpness?

Reply:
not sure - which one is correct?

Reply:
After taking again a look into Gneiting et. al. "Probabilistic forecasts, calibration and sharpness" [1]. I would say that this is sharpness. 

[1] https://sites.stat.washington.edu/raftery/Research/PDF/Gneiting2007jrssb.pdf

Reply:
renamed